homeassistant/components/cast/config_flow.py

```diff
@@ -139,9 +145,9 @@ async def async_step_basic_options(self, user_input=None):
                 self.hass.config_entries.async_update_entry(
                     self.config_entry, data=self.updated_config
                 )
-                return self.async_create_entry(title="", data=None)
+                return self.async_create_entry(title="", data={})
```

Comment:
According to `mypy`, `self.async_create_entry` doesn't allow data to be `None`

https://github.com/home-assistant/core/blob/34680becaac608a7ea6297f50f3c2a7cf7189160/homeassistant/data_entry_flow.py#L730-L737